docs/api/figures.rst

```diff
@@ -0,0 +1,79 @@
+
+.. code:: python
+
+    import matplotlib.pyplot as plt
+
+    from yellowbrick.features import FeatureImportances
+    from yellowbrick.classifier import ConfusionMatrix, ClassificationReport, ROCAUC
+    from sklearn.linear_model import LogisticRegression
+
+    fig, axes = plot.subplots(2, 2)
+
+    model = LogisticRegression()
+    visualgrid = [
+        FeatureImportances(ax=axes[0][0]),
+        ConfusionMatrix(model, ax=axes[0][1]),
+        ClassificationReport(model, ax=axes[1][0]),
+        ROCAUC(model, ax=axes[1][1]),
+    ]
+
+    for viz in visualgrid:
+        viz.fit(X_train, y_train)
+        viz.score(X_test, y_test)
+        viz.finalize()
+
+    plt.show()
+
+This is a common use case and we're working on the idea of "visual pipelines" to support this type of development because, for machine learning, users generally want a suite of visualizers or a report, not just a single visualization. The API requirement to support this has therefore been that visualizers use the ``ax`` object passed to them and not ``plt``. If the user does not pass a specific ``ax`` then the global current axes is used via ``plt.gca``. Generally, visualizers should behave as though they are a plot that as part of a larger figure.
+
+Visualizers are getting more complex, however, and some are becoming multi-axes plots in their own right. For example:
+
+- The ResidualsPlot has a scatter plot axes and a histogram axes
+- The JointPlot has a scatter plot and two histogram axes
+- Data driven scatter plot axes often have colorbar axes
+- The PCA plot has scatter plot, color bar, and heatmap axes
+- The confusion matrix probability histogram is a grid of axes for each class pair
+- The ICDM has an inset axes that acts as a dynamic legend
+
+Although it would have been easier to simply embed the figure into the visualizer and use a ``GridSpec`` or other layout tool, the focus on ensuring visualizers are individual plots that wrap an Axes has made us bend over backward to adjust the plot inside of the axes area that was originally supplied, primarily by using ``make_axes_locateable``, which is part of the AxesGrid toolkit.
+
+Generally, it appears that the `AxesGrid Toolkit <https://matplotlib.org/mpl_toolkits/axes_grid/users/overview.html#imagegrid>`_ is the right tool for Yellowbrick - many of the examples shown are similar to the things that Yellowbrick is trying to do. However, this package is not fully documented with examples and some helper utilities that would be useful, for example the ``ImageGrid``, still require a ``figure.Figure`` object.
+
+At this point we are left with some important questions about Yellowbrick's development roadmap:
+
+1. Like Seaborn, should YB have to classes of visualizer, one that wraps an axes and one that wraps a figure?
```

Comment:
These questions should be posted to our listserv, twitter,  
numfocus listserv

Reply:
I've added a para at the top that makes this more like an "open letter". Only the change in `docs/matplotlib.rst` is really important; should we include this in our docs to send around to the listservs, or should we just take the draft copy and send it ourselves?

Reply:
not sure I  understand what you mean "include in docs to send" versus "a draft copy"

Reply:
Yep, sorry I can see how that was unclear. I mean should we merge this PR into our docs and send around a link vs do not merge it into our docs and send it directly (thus the PR acts as a draft).

Reply:
I think it should be merged into the docs and a link sent around

Reply:
Ok, in that case - let me know if you need anything for this to be approved and we'll get it merged in!